docs/apache-airflow/executor/kubernetes.rst

```diff
@@ -214,7 +214,9 @@ To get task logs out of the workers, you can:
 Comparison with CeleryExecutor
 ------------------------------
 
-In contrast to CeleryExecutor, KubernetesExecutor does not require additional components such as Redis and Flower, but does require access to Kubernetes cluster.
+In contrast to CeleryExecutor, KubernetesExecutor does not require additional components such as Redis, but does require access to Kubernetes cluster.
+
+Also monitoring the Pods can be done with the built-in Kubernetes monitoring. For Celery you might want to use tools like Flower to monitor running Celery workers.
```

Comment:
```suggestion
Also monitoring the Pods can be done with the built-in Kubernetes monitoring.
```

Probably better to add that to the celery page instead? Seems out of place here to me. Maybe even bring this into the paragraph above?

Reply:
Crossed my mind too :)

Reply:
The less we mention Flower, the better. Removed it from here simply.